### PR TITLE
[Xtro] Update xtro after GameKit.

### PR DESCRIPTION
**Pull request by mandel-macaque**
WE had a mismatch between what was tested and what was compiled which
resulted in us missing this.

### CODE REVIEW

**Review by chamons — APPROVED** [review]

**Review by dalexsoto — APPROVED** [review]

**Comment by stephen-hawley** [comment]
👀

**Review by SotoiGhost — APPROVED** [review]

File: tests/xtro-sharpie/common-GameKit.ignore [hunk 1]
```diff
@@ -12,6 +12,9 @@
 !missing-selector! GKLeaderboardEntry::rank not bound
 !missing-selector! GKLeaderboardEntry::score not bound
 
+# Xcode1 4 beta 4
+!deprecated-attribute-wrong! GKBasePlayer::playerID has 16.0 not 13.0 on [Deprecated] attribute
```
Comment (SotoiGhost) [inline tests/xtro-sharpie/common-GameKit.ignore:16]: Should these changes be added to the `api-annotations-dotnet/common-GameKit.ignore` file

File: tests/xtro-sharpie/macOS-GameKit.ignore [hunk 2]
```diff
@@ -21,3 +21,6 @@
 
 # API removed by Apple, marked as deprecated
 !unknown-native-enum! GKAuthenticationType bound
+
+# the version is not 150.15
```
Comment (tj-devel709) [inline tests/xtro-sharpie/macOS-GameKit.ignore:25]:
```suggestion
# the version is not 10.15
```

**Review by tj-devel709 — APPROVED** [review]
Tiny comment change suggestion but otherwise LGTM

File: tests/xtro-sharpie/api-annotations-dotnet/macOS-GameKit.ignore [hunk 3]
```diff
@@ -6,3 +6,6 @@
 # This selector does not exist in macOS, confirmed by intro.
 !missing-selector! GKLeaderboardSet::loadImageWithCompletionHandler: not bound
 
+# the version is not 150.15
```
Comment (tj-devel709) [inline tests/xtro-sharpie/api-annotations-dotnet/macOS-GameKit.ignore:9]:
```suggestion
# the version is not 10.15
```

**Pull request merged**